static/js/ktl-component/footer/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import GlobalFooter from '@jetbrains/kotlin-web-site-ui/dist/footer.js';
+import '@jetbrains/kotlin-web-site-ui/dist/footer.css';
```

Comment:
Is not possible case when footer/header not will have styles when js is disabled?

Reply:
There is nothing to worry about. We use the "ExtractCssPlugin" that extracts all styles into a separate css file ("global.css").